mlflow/store/model_registry/abstract_store.py

```diff
@@ -458,3 +458,74 @@ def _await_model_version_creation_impl(self, mv, await_creation_for, hint=""):
                 f"{entity_type} version creation failed for {entity_type.lower()} name: {mv.name} "
                 f"version: {mv.version} with status: {mv.status} and message: {mv.status_message}"
             )
+
+    # Prompt management methods
+
+    @abstractmethod
+    def create_prompt(self, name: str, description: Optional[str] = None, tags: Optional[dict[str, str]] = None):
+        """
+        Create a new prompt (metadata only, no initial version).
+
+        Args:
+            name: Name of the prompt.
+            description: Description of the prompt.
+            tags: Dictionary of tag key-value pairs.
+
+        Returns:
+            A :py:class:`mlflow.store._unity_catalog.registry.prompt_info.PromptInfo` object.
+        """
+
+    @abstractmethod
+    def search_prompts(
+        self,
+        filter_string: Optional[str] = None,
+        max_results: Optional[int] = None,
+        order_by: Optional[list[str]] = None,
+        page_token: Optional[str] = None,
+        catalog_name: Optional[str] = None,
+        schema_name: Optional[str] = None,
+    ):
```

Comment:
```suggestion
    ) -> ...:
```

let's add the return type hint.

Reply:
+1. @rohitarun-db can we make sure to add return type hints?

Reply:
Done 👍